Core/NoOpRunner.Core/LoggingLevel.cs

```diff
@@ -7,6 +7,8 @@ namespace NoOpRunner.Core
         None = 0,
         Trace = 1,
         Pattern = 2,
-        Other = 4
+        Other = 4,
+        State = 5,
+        Iterator = 6
```

Comment:
Separate flags for separate patterns is probably a smart idea, easier to tweak what is shown, but you need to set the values properly (to powers of 2, so 8 and 16) for the flags to work properly

Reply:
Fixed